lib/cf/broker/post-usage.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+instance_id='8b97d2da-9710-49ae-89d9-5a3773811592'
+CLIENT_ID="abacus-rp-${instance_id}"
+CLIENT_SECRET='Huijeu6MM4'
```

Comment:
Hard-coding the secret does not seem like a good idea for two reasons:
* not all installations will have the same secret
* security